### DISCUSSION TITLE
How long after being approved do i get access to student benefits?

### DISCUSSION THREAD

**Discussion by Knownxc** [community/community#103798] [GitHub Education]
I've recently been approved to GitHub student but I still haven't gotten access to GitHub education. Can someone assist me and/or give me more information?!

[Screenshot_2024-02-04-13-31-07-900-edit_com.android.chrome.jpg](https://github.com/community/community/assets/146118693/64609de6-493a-49c4-9a6f-5115c3c016c3)

**Comment by Cauatn** [comment]
I believe it took up to two weeks to gain access to the benefits, as was the case for me, but I've had acquaintances who managed to do it in one or two days. 
For more information, you can check the GitHub Study Benefits https://education.github.com/globalcampus/student

**Reply by Knownxc** [reply]
Thx

**Reply by Cauatn** [reply]
I suggest marking this discussion as answered; sometimes, it can be bothersome to receive notifications from people responding to it in the community from time to time.
If the option doesn't appear, change the category that should resolve it to close the discussion. 😅

**Comment by mishmanners** [comment]
Once you are approved, you'll be notified as soon as the benefits are available.
